modules/utxo-bin/src/commands.ts

```diff
@@ -7,25 +7,44 @@ import * as utxolib from '@bitgo/utxo-lib';
 
 import { Parser, ParserArgs } from './parse';
 import { formatTree } from './format';
+import {
+  fetchOutputSpends,
+  fetchPrevOutputs,
+  fetchPrevOutputSpends,
+  fetchTransactionHex,
+  fetchTransactionStatus,
+} from './fetch';
 
 type Args = {
   network: string;
-  path: string;
+  stdin: boolean;
+  path?: string;
+  txid?: string;
   all: boolean;
   format: 'tree' | 'json';
+  fetchAll: boolean;
+  fetchStatus: boolean;
+  fetchInputs: boolean;
+  fetchSpends: boolean;
 } & ParserArgs;
 
 export function getParser(argv: yargs.Arguments<Args>): Parser {
   return new Parser(argv.all ? Parser.PARSE_ALL : argv);
 }
 
 export const cmdParse = {
-  command: 'parse <path>',
+  command: 'parse [path]',
   describe: 'display transaction components in human-readable form',
 
   builder(b: yargs.Argv<unknown>): yargs.Argv<Args> {
     return b
-      .option('path', { type: 'string', default: '-' })
+      .option('path', { type: 'string', nargs: 1, default: '' })
```

Comment:
Why is `nargs` and `default` set here? Would this option just default to `undefined` without it?

Reply:
This is to make `yargs parse -` actually parse stdin.  Without this the, `-` argument is ignored: https://github.com/yargs/yargs/issues/1312#issuecomment-940633564